### PR TITLE
Update + promote Jitsi Meet to recommended VoIP now it has E2EE

### PR DESCRIPTION
**Pull request by lrq3000**
## Description

Resolves: #1944

#### Check List 

- [x] I understand that by not opening an issue about a software/service/similar addition/removal, this pull request will be closed without merging.

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytools/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [x] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software

* Netlify preview for the mainly edited page: https://deploy-preview-1945--privacytools-io.netlify.app/software/real-time-communication/#voip

* Code repository of the project (if applicable): https://github.com/jitsi/jitsi-meet

### CODE REVIEW

**Comment by lrq3000** [comment]
Please put this PR on hold for the moment because of issues with Firefox. Please see #1944 for more info.

**Comment by lrq3000** [comment]
I have demoted Jitsi Meet to 2nd place after Linphone, and added a warning label "Experimental E2EE", similarly to Rocket.chat. Since Rocket.chat also has an experimental E2EE, I think it would also be reasonable to upgrade Jitsi Meet to a recommendation with a similar warning.

Up to the reviewers and others to decide whether to merge as-is or keep on hold until the issue with Firefox is resolved :-)

File: assets/img/svg/3rd-party/jitsi-meet-blue.svg [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+      </g>
+    </g>
+    <path
+       style="fill:#ffffff"
+       d=""
+       id="path3618"
+       inkscape:connector-curvature="0" />
+  </g>
+</svg>
```
Comment (dngray) [inline assets/img/svg/3rd-party/jitsi-meet-blue.svg:182]: Please see https://github.com/lrq3000/privacytools.io/pull/1

Comment (lrq3000) [reply assets/img/svg/3rd-party/jitsi-meet-blue.svg]: Thank you, it's done now and I also updated my other PRs accordingly :-) Thanks a lot for the tutorial, very much appreciated!

**Pull request merged**